pkgs/development/libraries/openvino/default.nix

```
@@ -125,8 +129,8 @@ stdenv.mkDerivation rec {
     (cmakeBool "ENABLE_SAMPLES" false)
 
     # features
-    (cmakeBool "ENABLE_INTEL_CPU" true)
-    (cmakeBool "ENABLE_INTEL_GNA" true)
+    (cmakeBool "ENABLE_INTEL_CPU" (! stdenv.isAarch64))
+    (cmakeBool "ENABLE_INTEL_GNA" (! stdenv.isAarch64))
```

This probly also doesn't apply to other architectures so it probaly should only be trie when isx86 maybe even only on isx86_64 depending if this supports 32 bit or not

```suggestion
    (cmakeBool "ENABLE_INTEL_CPU" stdenv.isx86)
    (cmakeBool "ENABLE_INTEL_GNA" stdenv.isx86)
```

---

Done. Thanks for the suggestion!